ci-operator/step-registry/cri-o/cgroupv2/e2e-crun/test/cri-o-cgroupv2-e2e-crun-test-commands.sh

```diff
@@ -26,7 +26,7 @@ cp "${CLUSTER_PROFILE_DIR}"/ssh-privatekey "${HOME}"/.ssh/google_compute_engine
 chmod 0600 "${HOME}"/.ssh/google_compute_engine
 cp "${CLUSTER_PROFILE_DIR}"/ssh-publickey "${HOME}"/.ssh/google_compute_engine.pub
 
-latest="v3.5.4"
+latest="v3.5.6"
```

Comment:
try this:
`latest=$(wget -q -O- https://api.github.com/repos/coreos/etcd/releases/latest | jq -r '.name')`

Reply:
In case jq is missing then
`latest=$(wget -q -O- https://api.github.com/repos/coreos/etcd/releases/latest |  grep '"tag_name":' | sed -E 's/.*"([^"]+)".*/\1/')`